src/plugins/deps/geomap-lib.js

```diff
@@ -96,1063 +93,756 @@ var componentName = "wb-geomap",
 					geolocBtn: i18n( "geo-geoloc-btn" ),
 					geolocFail: i18n( "geo-geoloc-fail" ),
 					geolocUncapable: i18n( "geo-geoloc-uncapable" ),
-					geoLgndGrphc: i18n( "geo-lgnd-grphc" )
+					geoLgndGrphc: i18n( "geo-lgnd-grphc" ),
+					dismiss: i18n( "dismiss" )
 				};
 			}
 
 			// Class-based overrides - use undefined where no override should occur
 			overrides = {
-				useScaleLine: className.indexOf( "scaleline" ) !== -1 ? true : undefined,
-				useMousePosition: className.indexOf( "position" ) !== -1 ? true : undefined,
-				useLegend: className.indexOf( "legend" ) !== -1,
-				useTab: className.indexOf( "tab" ) !== -1,
-				useMapControls: className.indexOf( "static" ) !== -1 ? false : true,
-				useGeocoder: className.indexOf( "geocoder" ) !== -1 ? true : false,
-				useGeolocation: className.indexOf( "geolocation" ) !== -1 ? true : false,
-				useAOI: className.indexOf( "aoi" ) !== -1 ? true : false
+				useScaleLine: className.indexOf( " scaleline " ) !== -1 ? true : undefined,
```

Comment:
if the spaces are to avoid the substring, you could just convert className to an array first and then use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/indexOf